en/firefox/features/index.ftl

```diff
@@ -0,0 +1,114 @@
+
+# Obsolete string, replaced in firefox/features/shared.ftl
+features-index-password-manager = Password Manager
+features-index-access-all-passwords = { -brand-name-firefox-lockwise } lets you access all the passwords you’ve saved in Firefox — and it’s free.
+
+# Obsolete string, replaced in firefox/features/shared.ftl
+features-index-balanced-memory-usage = Balanced memory usage
+features-index-just-enough = { -brand-name-firefox } uses just enough memory to create a smooth experience so your computer stays responsive to other tasks.
+
+# Obsolete string, replaced in firefox/features/shared.ftl
+features-index-browse-faster = Browse faster
+features-index-use-less-memory = { -brand-name-firefox } uses less memory than { -brand-name-chrome }, so your other programs can keep running at top speed.
+
+# Obsolete string
+features-index-firefox-product-benefits = { -brand-name-firefox } Product Benefits
+
+# Obsolete string
+features-index-open-source = Open source
+
+# Updated string
+features-index-open-source-minds = Open source. Open minds.
+features-index-mozilla-creates = { -brand-name-mozilla } creates powerful web tech for everyone.
+
+# Obsolete string
+features-index-were-always-transparent = We’re always transparent.
+
+# Obsolete string
+features-index-see-what-makes-us-different = See what makes us different
+features-index-by-non-profit-mozilla = By non-profit, { -brand-name-mozilla }
+features-index-on-a-mission = On a mission to keep the internet open and accessible to all.
+features-index-keep-corporate-power = Keep corporate power in check
+features-index-independent-browser = { -brand-name-firefox } is the only major independent browser.
+
+# Obsolete string
+features-index-protecting-the-health-of-the = Protecting the health of the internet.
+features-index-read-mozillas-mission = Read { -brand-name-mozilla }’s mission
+
+# Obsolete string
+features-index-protect-your-rights = Protect your rights
+
+# Obsolete string
+features-index-we-help-keep-corporate-powers = We help keep corporate powers in check.
+features-index-choose-independence = Choose independence
+
+# Obsolete string
+features-index-limited-data-collection = Limited data collection
+features-index-opted-in-to-privacy-so-you = Opted-in to privacy, so you can browse freely.
+features-index-read-our-privacy-policy = Read our privacy policy
+
+# Obsolete string
+features-index-more-private = More private
+
+# Updated string
+features-index-private-by-default = Private by default
+features-index-enhanced-tracking = Enhanced tracking protection
+features-index-we-dont-sell-access-to-your = We don’t sell access to your online data. Period.
+features-index-get-firefox-for-privacy = Get { -brand-name-firefox } for privacy
+features-index-firefox-vs = { -brand-name-firefox } vs. other browsers
+features-index-stack-up = See how { -brand-name-firefox } stacks up against other popular browsers.
+features-index-compare-browsers = Compare browsers
```

Comment:
# Obsolete string, replaced in firefox/features/shared.ftl for `Compare browsers`?